project/src/com/google/daggerquery/executor/models/Graph.java

```diff
@@ -30,6 +30,11 @@
    */
   ImmutableSet<String> getDependencies(String node);
 
+  /**
+   * Returns all node's ancestors.
```

Comment:
I chose this name because I thought it was quite common in graph theory, for example there is an LCA problem called [Least Common Ancestor](https://www.hackerrank.com/topics/lowest-common-ancestor), and so on.

Reply:
I have clarified [what an ancestor is](https://github.com/googleinterns/dagger-query/pull/43/commits/c58f408b6947d79ca43ec11e4cc47f4830557c35) in this commit